src/main/java/org/folio/marc4ld/service/marc2ld/field/ResourceProcessorImpl.java

```diff
@@ -71,6 +71,13 @@ private void additionalMapping(Resource resource,
       .ifPresent(m -> m.map(new MarcData(dataField, controlFields), resource));
 
     resource.setId(hashService.hash(resource));
+    setLabel(resource);
```

Comment:
I think that we should set ```label``` before setting the ```id```. Because according to fingerprint-rules ```label``` property is the part of  the ```id```.